src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsFetchingSpec.groovy

```diff
@@ -1701,6 +1701,38 @@ class PriceFloorsFetchingSpec extends PriceFloorsBaseSpec {
         assert bidderRequest.ext?.prebid?.floors?.location == FETCH
     }
 
+    def "PBS should set floors skipped flag = false when account with enabled fetch"() {
```

Comment:
`PBS should always populate floors skipped flag when floors are enabled for account`